### PR TITLE
Song list in album

### PR DESCRIPTION
**Pull request by Ailey25**
Clicking into an album shows a list of all songs in that album
Changed AlbumData to a class with static methods
Changed MediaPlayer to static to ensure only one song plays at a time

### CODE REVIEW

File: app/src/main/java/com/aileyzhang/musicapp/activities/MainActivity.java [hunk 1]
```diff
@@ -29,8 +30,8 @@
         ActivityCompat.OnRequestPermissionsResultCallback {
 
     private static final int SDCARD_PERMISSION_REQUEST_CODE = 1;
-
-    private ViewPager mViewPager;
+    public static MainActivityContentAdapter mMainActivityContentAdapter;
```
Comment (jiesiren) [inline app/src/main/java/com/aileyzhang/musicapp/activities/MainActivity.java]: Why is this static? owo

File: app/src/main/java/com/aileyzhang/musicapp/SongData.java [hunk 2]
```diff
@@ -54,15 +48,52 @@ private void loadAllSongs(Context context) {
...
-    private void loadSongsFromAlbum(String albumID) {
+    public static ArrayList<Song> getSongsInAlbum (Context context, String albumName) {
+        ArrayList<Song> songs = new ArrayList<>();
+        String[] projection = new String[]{
+                MediaStore.Audio.AudioColumns.DATA,
+                MediaStore.Audio.AudioColumns.TITLE,
+                MediaStore.Audio.ArtistColumns.ARTIST,
+                MediaStore.Audio.AudioColumns.ALBUM,};
+        String selection = MediaStore.Audio.AudioColumns.ALBUM + "=?";
+        String[] selectionArgs = new String[] {"" + albumName};
```
Comment (jiesiren) [inline app/src/main/java/com/aileyzhang/musicapp/SongData.java:68]: Why is there a `"" + <somethingThatsAlreadyAString>`, and also do we need to query by albumName? Does album ID not work or something? =w=

Comment (Ailey25) [reply app/src/main/java/com/aileyzhang/musicapp/SongData.java:68]: I'll change that. albumID doesn't work. Think it's unique for the same albums as well. The cursor returns null for me when I use albumID

File: app/src/main/java/com/aileyzhang/musicapp/activities/MainActivity.java [hunk 3]
```diff
@@ -29,8 +30,8 @@
         ActivityCompat.OnRequestPermissionsResultCallback {
 
     private static final int SDCARD_PERMISSION_REQUEST_CODE = 1;
-
-    private ViewPager mViewPager;
+    public MainActivityContentAdapter mMainActivityContentAdapter;
+    public ViewPager mMainViewPager;
```
Comment (jiesiren) [inline app/src/main/java/com/aileyzhang/musicapp/activities/MainActivity.java:34]: sorry to nitpick again lmao, can these be private? Best to restrict access to how much is being used, or access level intended by design 👌

**Review by jiesiren — APPROVED** [review]

**Pull request merged**